vdb-bench-assembly/plugins/vdb-bench-widgets/fileImportControl.js

```diff
@@ -71,8 +71,8 @@
                 //
                 // Valid formats currently implemented
                 //
-                var validFormats = ['ZIP', 'XML', 'DDL'];
-                var documentType = myFile.name.substring(myFile.name.lastIndexOf(SYNTAX.DOT) + 1).toUpperCase();
+                var validFormats = ['zip', 'xml', 'ddl'];
+                var documentType = myFile.name.substring(myFile.name.lastIndexOf(SYNTAX.DOT) + 1);
```

Comment:
If validFormats changes to lowercase then the substring(....) line below it needs to have toLowerCase() on the end since its possible to have a file ending in uppercase or lowercase file extension and still be valid.
This probably needs to be modified anyway since the validFormats are { zip, -vdb.xml, ddl, tds }, although whether tds needs to be included at this point is hard to say.